packages/apex-node/src/logs/logService.ts

```diff
@@ -14,10 +14,14 @@ import {
 import { createFile } from '../utils';
 import { nls } from '../i18n';
 import * as path from 'path';
+import * as dayjs from 'dayjs';
 import { AnyJson } from '@salesforce/ts-types';
+import { QueryResult } from '../utils/types';
 
+const DEFAULT_DEBUG_LEVEL_NAME = 'SFDC_DevConsole';
```

Comment:
since there's many constants now, we can probably move them into their own `contstants.ts` file within the `logs` module